x-pack/functionbeat/magefile.go

```diff
@@ -183,6 +183,29 @@ func BuildPkgForFunctions() error {
 	return nil
 }
 
+// TestGCPFunctions are used by Travis to test if the GCP functions can be built with
```

Comment:
As it runs in both Travis and Jenkins:

```suggestion
// TestGCPFunctions are used by the CI to test if the GCP functions can be built with
```